test/unit/spec/RefreshAuthState_spec.js

```diff
@@ -36,18 +36,21 @@ function (Q, Okta, OktaAuth, Util, Beacon, FormView, Expect,
     Util.registerRouter(router);
     Util.mockRouterNavigate(router);
     Util.mockJqueryCss();
-    return Q({
-      router: router,
-      beacon: beacon,
-      form: form,
-      ac: authClient,
-      setNextResponse: setNextResponse
+    setNextResponse(resEnroll);
+    return Util.mockIntrospectResponse(router, resEnroll).then(function () {
+      return Q({
+        router: router,
+        beacon: beacon,
+        form: form,
+        ac: authClient,
+        setNextResponse: setNextResponse
+      });
     });
   }
 
   Expect.describe('RefreshAuthState', function () {
 
-    itp('redirects to PrimaryAuth if authClient does not need a refresh', function () {
+    /*itp('redirects to PrimaryAuth if authClient does not need a refresh', function () {
```

Comment:
@nikhilvenkatraman-okta did we forget to fix the test?

Reply:
ah, looks like we missed to update this test.

Reply:
Fixed [here](https://github.com/okta/okta-signin-widget/pull/899)